cwms-data-api/src/main/java/cwms/cda/data/dao/project/ProjectDao.java

```diff
@@ -301,27 +316,44 @@ private static String buildTableQuery(@Nullable String projectIdMask, @Nullable
         return sql;
     }
 
+    /**
```

Comment:
for the code right above this,
private static String buildTableQuery(@Nullable String projectIdMask, @Nullable String office,
                                          String cursorOffice, String cursorProjectId) {
In terms of clarity, you could swap the Strings to booleans to insure that the Strings are not used in the construction of the query?

Reply:
done